torch/fx/graph.py

```diff
@@ -94,6 +94,9 @@ def __init__(self):
         self._unassociated_names = set()
         self._used_names: Dict[str, int] = {}
 
+        self._illegal_char_regex = re.compile('[^0-9a-zA-Z_]+')
```

Comment:
could jsut be module scoped (or a class constant) so that we don't even have to recompile between instances of `_Namespace`

Reply:
yeah it could be but i tend to avoid globals. if it becomes an issue we can look into it